bench-tps/src/main.rs

```diff
@@ -80,7 +82,30 @@ fn main() {
         exit(1);
     }
 
-    let (keypairs, keypair_balance) = if read_from_client_file {
+    let libra_mint_keypair = if let Some(libra_mint_keypair_file) = libra_mint_keypair_file {
+        if !use_move {
+            eprintln!("No keypair file needed if move is not enabled");
+            exit(1);
+        }
+
+        Some(Arc::new(
+            read_keypair(&libra_mint_keypair_file).unwrap_or_else(|err| {
+                eprintln!(
+                    "{}: Unable to open keypair file: {}",
+                    err, libra_mint_keypair_file
+                );
+                exit(1);
+            }),
+        ))
+    } else {
+        if use_move {
+            eprintln!("Keypair file needed if move is enabled, use --libra-mint-keypair argument");
```

Comment:
You can get Clap to do this for you.  Just tell Clap that these two args require each other